static/templyte/src/app/tutorial/tutorial.component.html

```diff
@@ -0,0 +1,82 @@
+<mat-toolbar class="mat-elevation-z3">
+    <nav class="navbar">
+        <ul class="nav navbar-right">
+            <li>
+                <button mat-button (click)="goHome()">Home</button>
+            </li>
+            <li>
+                <button mat-button (click)="createTemplate()">Create Template</button>
+            </li>
+            <li>
+                 <button mat-button disabled>Tutorial</button>
+            </li>
+            <li>
+                <button mat-button (click)="myTemplates()">My Templates</button>
+            </li>
+            <li>
+                <button mat-button (click)="logout()">Logout</button>
+            </li>
+        </ul>
+    </nav>
+</mat-toolbar>
+
+<html>
+<head>
+
+<title>Page Title</title>
+</head>
+<body>
+<h1>Create a Template</h1>
+
+<ol type="1">
+  <li>Navigate to the "Create Template" page from the navigation bar at the top of each page</li>
+  <img src="assets/img/tutorial/menu.jpg" alt="Menu" width="533" height="44" >
+  <li>Provide a name and description of your template</li>
+  <img src="assets/img/tutorial/templateNameAndDescr.png" alt="Menu" width="761" height="105" >
+  <li>Select the zip file of the template</li>
+  <li>Provide Names and Descriptions of Template Variables</li>
+  <img src="assets/img/tutorial/templateVariables.png" alt="Menu" width="765" height="223" >
+  <li>Provide Tags</li>
+  <img src="assets/img/tutorial/templateTags.png" alt="Menu" width="767" height="193" >
+  <li>Provide File Endings</li>
+  <img src="assets/img/tutorial/fileEndings.png" alt="Menu" width="760" height="180" >
+  <li>Submit template</li>
+</ol>
```

Comment:
Tutorial doesn't cover how to create the content of the zip file - most of the other stuff is self explanatory, so it's not as important. Currently, there is no way for a user to know how to actually build a template before uploading.